pym/gentoolkit/eclean/search.py

```diff
@@ -573,7 +573,7 @@ def findPackages(
     settings = var_dbapi.settings
     bin_dbapi = portage.binarytree(pkgdir=pkgdir, settings=settings).dbapi
     if "force_reindex" in signature(bin_dbapi.bintree.populate).parameters:
-        bin_dbapi.bintree.populate(force_reindex=True)
+        bin_dbapi.bintree.populate(force_reindex=True, invalid_errors=False)
```

Comment:
Could test if invalid_errors is supports, like:
```python
    pop_sig_params = signature(bin_dbapi.bintree.populate).parameters
    populate_kwargs = {}
    if "invalid_errors" in pop_sig_params:
        populate_kwargs["invalid_errors"] = True
    bin_dbapi.bintree.populate(force_reindex=True, **populate_kwargs)
```

Reply:
Thanks @zmedico I forgot again :-(